src/DelegateInterface.php

```diff
@@ -8,11 +8,11 @@
 interface DelegateInterface
 {
     /**
-     * Dispatch the next available middleware and return the response.
+     * Process a request and return the response.
```

Comment:
This commentary is still not correct. A delegate is deferring processing to the next available middleware. It is not, in and of itself, processing anything.